lcc_comchain_base/datamodel/comchain.py

```diff
@@ -0,0 +1,71 @@
+from marshmallow import fields
+
+from odoo.addons.datamodel.core import Datamodel
+from odoo.addons.datamodel.fields import NestedModel
+
+
+class AddressesInfo(Datamodel):
```

Comment:
can't find a place where this would be used. Do you want to keep it for coherence between modules ?